pyproject.toml

```diff
@@ -101,7 +101,7 @@ docs = [
   # Needed for notebooks/HydroShare_integration.ipynb
   # See: https://github.com/CSHS-CWRA/RavenPy/pull/326
   # "hsclient",
-  "intake <2.0.0",
+  "intake",
```

Comment:
FYI, I don't think this will change anything as `intake-esm` is also pinning older `intake`, see https://github.com/Ouranosinc/pavics-jupyter-env-issues/issues/17#issuecomment-2555976803

Reply:
I'm trying to see what can be re-pinned here. I get the impression that there are more than a few badly pinned deps. It's hard enough to get it running atm.